apps/web/src/hooks/useOptInRedirect.ts

```diff
@@ -13,29 +13,26 @@ export const useOptInRedirect = () => {
   const { pathname } = useLocation();
   const isNewDashboardEnabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_NEW_DASHBOARD_ENABLED);
   const { status, isLoaded, redirectToNewDashboard } = useNewDashboardOptIn();
-  const isUserOptedIn = isLoaded && status && status === NewDashboardOptInStatusEnum.OPTED_IN;
-  const dashboardV2HostName = window.location.hostname.includes('dashboard-v2');
```

Comment:
this condition is not needed - we should redirect to new dashboard when legacy web is deployed on both `dashboard-v2` and `dashboard` domains